metaflow/metaflow_config.py

```diff
@@ -45,6 +45,13 @@ def from_conf(name, default=None):
 DEFAULT_DATASTORE = from_conf('METAFLOW_DEFAULT_DATASTORE', 'local')
 DEFAULT_METADATA = from_conf('METAFLOW_DEFAULT_METADATA', 'local')
 
+##
+# Per run configuration
+###
+KFP_SDK_NAMESPACE = from_conf('KFP_SDK_NAMESPACE')
```

Comment:
What should the default of this be? `kubeflow` ?

Reply:
Done.

Reply:
@talebzeghmi `KFP_SDK_API_NAMESPACE` should default to `kubeflow`. See the change 2 lines below.

Reply:
Where is this done for KFP_SDK_NAMESPACE?

Reply:
There is no default for SDK namespace.

Reply:
Is it not "kubeflow" for a default installation of Kubeflow?

Reply:
This is the `namespace` on KFP, which you can see on the UI. For example, `aip-example`,  `talebz`, `hariharans`, etc. I can just put `aip-example` if you really want a default.

Reply:
Which you see in the UI is very ambiguous and not specific.  You mean to say that this is the namespace on which the pipeline will run.
The default for a Kubeflow installation is the "kubeflow" Kubeflow namespace.

Reply:
Correct, this refers to the namespace the KFP run is under, not the Kubeflow installation.

Reply:
And the default namespace for a default Kubeflow deployment is "kubeflow"